lookout/style/format/utils.py

```diff
@@ -14,19 +14,23 @@
 from lookout.style.format.files_filtering import filter_filepaths
 
 
-def prepare_files(folder: str, client: BblfshClient, language: str) -> Iterable[File]:
+def prepare_files(files_location: Union[str, Iterable[str]], client: BblfshClient,
```

Comment:
Better to name it just `files` then

Reply:
Waren can you please remove the original case (string) and insert `glob.glob(recursive=True)` to the usage places. So we should accept only an Iterable